### ISSUE TITLE
planemo lint should check <version_command> exists

### ISSUE THREAD

**Issue by peterjc** [galaxyproject/planemo#286]
I would like ``planemo lint`` to check for the presence of a ``<version_command>`` tag.

e.g. https://github.com/peterjc/pico_galaxy/issues/8

If this is missing it should at least be a warning, perhaps even an error?

We should also update https://github.com/galaxy-iuc/standards/blob/master/docs/best_practices/tool_xml.rst to make this a recommendation.

**Comment by nsoranzo** [comment]
I'd prefer a message at `info` level, surely not error. For pure Galaxy tools (not wrapping an underlying tool), I think it is perfectly ok to not have a `<version_command>`.

**Comment by peterjc** [comment]
Good point - making this an info/warning seems reasonable (not an error).
